datafusion/sqllogictest/test_files/distinct_on.slt

```diff
@@ -38,17 +38,17 @@ LOCATION '../../testing/data/csv/aggregate_test_100.csv'
 # Basic example: distinct on the first column project the second one, and
 # order by the third
 query TI
-SELECT DISTINCT ON (c1) c1, c2 FROM aggregate_test_100 ORDER BY c1, c3;
+SELECT DISTINCT ON (c1) c1, c2 FROM aggregate_test_100 ORDER BY c1, c3, c9;
```

Comment:
I think changing the order by changes the results because DISTINCT ON basically picks the first values for each of the distinct values (of c1) in this case

Reply:
Indeed